pkg/api/handlers/assistants.go

```diff
@@ -242,17 +242,22 @@ func (a *AssistantHandler) ListCredentials(req api.Context) error {
 
 func (a *AssistantHandler) Events(req api.Context) error {
 	var (
-		runID  = req.Request.Header.Get("Last-Event-ID")
+		follow = req.URL.Query().Get("follow") == "true"
+		runID  = req.URL.Query().Get("runID")
 		thread v1.Thread
 	)
 
+	if runID == "" {
+		runID = req.Request.Header.Get("Last-Event-ID")
+	}
+
 	if err := req.Get(&thread, req.PathValue("thread_id")); err != nil {
 		return err
 	}
 
 	_, events, err := a.events.Watch(req.Context(), req.Namespace(), events.WatchOptions{
-		Follow:        true,
-		History:       runID == "",
+		Follow:        follow,
+		History:       true,
```

Comment:
I have definitely used passing a `runID` to get the events for only that run. Is there a reason this is changing?

Reply:
I believe that's using the `/api/runs/{id}/events` endpoint, which I think is only for admins. Also the response I got back from that endpoint wasn't in the format I needed to display events. I can double check though.

Reply:
Are you saying that leaving this as-is and passing a `runID` (so not getting any history) breaks something?

Reply:
Spoke with @thedadams in a different channel and we're going to add another query parameter for setting the history field here.

Reply:
I added the `history` query parameter.